templates_ark/1_Segment_Image_Data.ipynb

```diff
@@ -41,6 +41,25 @@
     "If you're interested in directly interfacing with Google Drive, consult the documentation [here](https://ark-analysis.readthedocs.io/en/latest/_rtd/google_docs_usage.html)."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    tiff_dir = os.path.join(example_dataset_path, "input_data")
</pre><p>I don't think we need a separate <code>input_data</code> folder within <code>image_data</code>,the point is that <code>image_data</code> looks exactly the same as it will coming off toffy, containing only folders of TIFs    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Shouldn't it imitate what's in toffy?
![](https://user-images.githubusercontent.com/13770365/189005441-62df7b17-fca6-4317-bf62-528baa4207b2.png)

So that input_data would imitate 20220101_run_1_tonsil_1?

Reply:
No, each of those sub-folders are an individual FOV with 40 channels of TIF data, equivalent to the sub-folders within input_data

Reply:
Ah, gotcha. Makes sense.